lib/Parse/ParseExpr.cpp

```diff
@@ -1136,8 +1139,17 @@ Parser::parseExprPostfixSuffix(ParserResult<Expr> Result, bool isExprBasic,
 
       // Handle "x.<tab>" for code completion.
       if (Tok.is(tok::code_complete)) {
-        if (CodeCompletion && Result.isNonNull())
+        if (CodeCompletion && Result.isNonNull()) {
+          if (InSwiftKeyPath)
```

Comment:
For clarity, please put `{}` braces around these since they're split on several lines and it's important to know that the `completeDoctExpr` is not part of the `else`.